### DISCUSSION TITLE
What is the name of the vinegar specifically made from Trebbiano grapes?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#536] [Galaxy Trivia] [answered]
**Bonus:** What is the most expensive vinegar on earth?

**Comment by peterbaccaro** [comment]
**Balsamic vinegar** is the vinegar made from white Trebbiano grapes.

**Comment by Veercodeprog** [comment, marked as answer]
Traditional Balsamic Vinegar of Modena, which is produced in the Modena region of Italy. It is made from the must of Trebbiano grapes and is aged in wooden barrels for at least 12 years, and some varieties can be aged for over 25 years.

**Comment by Veercodeprog** [comment]
gulabo
